.github/workflows/compat.yml

```diff
@@ -66,12 +66,12 @@ jobs:
       - name: Install dependencies
         run: pnpm install
 
-      - name: Install TypeScript ${{ matrix.ts }}
-        run: pnpm add typescript@${{ matrix.ts }}
-
       - name: Build
         run: pnpm build
 
+      - name: Install TypeScript ${{ matrix.ts }}
+        run: pnpm add typescript@${{ matrix.ts }}
+
```

Comment:
shifted the build and typescript version change steps

We always want to _build_ with the currently installed version of typescript, which is what we ship to end users, but we want to run the typing tests on different versions

Reply:
@kettanaito this change dismissed the previous review, but also removes the need to have any `ts-ignores` so I removed that final one